{{cookiecutter.project_slug}}/.github/workflows/pytest.yml

```diff
@@ -0,0 +1,17 @@
+name: Unit Tests 
+on: [push]
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    env:
+      SECRET_KEY: {{ "${{ secrets.SECRET_KEY }}" }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: pipx install pipenv
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+          cache: 'pipenv'
+      - run: pipenv install --dev --deploy
+      - run: pipenv run pytest --mccabe --cov={{ cookiecutter.project_slug }} -vv {{ cookiecutter.project_slug }}
+      - run: pipenv run coverage report --fail-under=20
```

Comment:
I like this coverage report and the 20% threshold 👍🏻 

Do you know how easy it is to add a coverage badge to the README? It would be cool to keep this readily visible to everyone to encourage more test-writing.

Reply:
I think I want to keep this as-is, but mccabe found 3 high complexity functions in the utils folder. I hadn't looked in there before. My eyes went 👀 
So I think for now leave mccabe on(?), but that means that during bootstrap tests will immediately fail until someone improves those functions.
opinions?